### DISCUSSION TITLE
How to avoid Symlinks in .output OR how to copy them?

### DISCUSSION THREAD

**Discussion by skoenfaelt** [nuxt/nuxt#23633] [Questions]
Our Gitlab CI builds the app correctly, but as we copy the .output folder the script crashes due the symlinks in the treeshaked node_modules folder in the .output folder.

I cant find any hint, how to avoid the generation of symlinks. If there is none setting, is there any guidance how to copy the contents of symlinks?

**Comment by TheAlexLichter** [comment]
Not sure if you can - https://github.com/unjs/nitro/blob/ae253596a847f764e275f7e0450ffdfcdcd8957e/src/rollup/plugins/externals.ts#L347-L355

Would be good to raise an issue in Nitro 👍🏻

**Reply by skoenfaelt** [reply]
I got it to work without symlinks
`robocopy "$SourcePath" "$DestinationDir" /e /njh /njs /ndl /nc /ns /sl`
/s or /sl does the trick
https://ss64.com/nt/robocopy.html

But now windows has issues due the now long paths. :D So I guess symlinks are important and I need to figure out how I can i pass them through the CI Compiler to a different server. But I will also look what I can achive with your findings. Thanks manniL

**Comment by skoenfaelt** [comment]
Got it to work.
If first we copy our output to an artifacts directory with 
`robocopy "$SourcePath" "$DestinationDir" /e /njh /njs /ndl /nc /ns /sl`

and to copy to the server, we use `robocopy $ArtifactsDir $DestinationDir /mir` 
*important /mir works differently than /move, also in use case of long windows paths*
